deploy/examples/common.yaml

```diff
@@ -793,9 +793,12 @@ rules:
     resources:
       - deployments/scale
       - deployments
+      - pods
```

Comment:
I'm confused... `pods` isn't part of the `apps` group?

Reply:
@rkachach @travisn

Reply:
@travisn @uhthomas sorry for the late response (I was on leave). I don't have access to a system to test but as far as I remember without these changes on rbac I wasn't able to list mgr deployments from the side-car container.

Reply:
@rkachach Thanks for confirming, I didn't get a chance to test without it anyway. Makes sense to keep it then.

Reply:
I don't get how. Pods are part of the core group (`""`). `apps/v` doesn't have a kind `Pod`.

Reply:
@rkachach Looking deeper, the [rook-ceph-mgr](https://github.com/rook/rook/blob/release-1.11/deploy/examples/common.yaml#L740-L745) role already has the ability for the mgr to update pods. As @uhthomas mentioned, the pod isn't part of the apps, so this rbac is not affecting anything. @rkachach Are you available to open a PR to remove this? I can help test if needed.

Reply:
@travisn of course, I can give it a try next week :+1: